### ISSUE TITLE
excessive cpu load(Infinite)

### ISSUE THREAD

**Issue by tsabbay** [xmldom/xmldom#324]
We noticed 100% cpu load (infinite) when the data is containing `&#8232;` (&#8232;)  Or similar codes for new line.
Example of code
```
const { DOMParser } = require('xmldom')

let xmlDocument = new DOMParser().parseFromString(documentFile)
```

Attached is a document, beware of parsing this as it's going to choke your instance.
[hell end line.docx](https://github.com/xmldom/xmldom/files/7169584/hell.end.line.docx)

Happy to provide more info.

**Comment by karfau** [comment]
Thank you for providing the test data. There are earlier similar reports but nobody was able to provide example data yet.

Do i understand correctly that the docx file should be read into a string and its content parsed? (I thought it's a zip file.)

**Comment by tsabbay** [comment]
Yes, took us time to identify this
Read the file to string and try parse it.

**Comment by karfau** [comment]
Hey there. I looked into it.
First of all as I thought, the docx file is basically a zip file, so xmldom can not "parse" it directly as your initial post suggested.
It contains the following structure:
```
├── [Content_Types].xml
├── docProps
│   ├── app.xml
│   └── core.xml
├── _rels
└── word
    ├── document.xml
    ├── fontTable.xml
    ├── _rels
    │   └── document.xml.rels
    ├── settings.xml
    ├── styles.xml
    ├── theme
    │   └── theme1.xml
    └── webSettings.xml

5 directories, 10 files
```
I searched the files for the character you mentioned, and only found it in `word/document.xml`.

Good news is: this is already fixed on master since we landed #314 
We are currently working on the last two issues that will be part of the 0.8.0 release that will contain this bugfix. You can [check on our progress in the related milestone](https://github.com/xmldom/xmldom/milestone/10)
